aten/src/ATen/cudnn/Descriptors.h

```diff
@@ -18,6 +18,8 @@
 #include <ATen/ops/empty.h>
 #endif
 
+#define RNNV8VERSION 8907
```

Comment:
Can this be an explicit flag to simplify the #if in the rest of the code? Something like
#define USE_CUDDN_V8_RNN_API ...